src/MySqlConnector/Logging/ConsoleLoggerProvider.cs

```diff
@@ -37,10 +38,10 @@ public void Log(MySqlConnectorLogLevel level, string message, object[] args = nu
 				sb.Append(m_name);
 				sb.Append('\t');
 
-				if (args == null)
+				if (args == null || args.Length == 0)
```

Comment:
Should probably fix [the docs](https://github.com/bgrainger/MySqlConnector/blob/62061f9ba5af1af2b0a374b7b657cb3a587e6d84/src/MySqlConnector/Logging/IMySqlConnectorLogger.cs#L23-L26) as well.